instrumentation/logger/lib/opentelemetry/instrumentation/logger/patches/logger.rb

```diff
@@ -38,10 +37,20 @@ def skip_instrumenting?
           end
 
           def severity_number(severity)
-            ::Logger::Severity.const_get(severity)
-          rescue NameError => e
-            OpenTelemetry.logger.warn(message: "Unable to coerce severity text #{severity} into severity_number. Setting severity_number to nil.", exception: e)
-            nil
+            case severity.downcase
+            when 'debug'
+              Opentelemetry::Proto::Logs::V1::SeverityNumber::SEVERITY_NUMBER_DEBUG
```

Comment:
I believe these constants aren't available in any of the current dependencies for this gem. Right now, they're part of the OTLP exporters. I don't think we want to require an exporter for some instrumentation. 

One potential workaround would be to update the draft API to have a set of constants mapping to these proto values. However, I'm sure there's other ways to solve this problem and I'd love to hear your ideas.

Reply:
yes, I think having a map of these in draft API would make it accessible to all bridges!